sdcm/keystore.py

```diff
@@ -30,6 +30,9 @@ def get_gcp_credentials(self):
     def get_scylladb_upload_credentials(self):
         return self._get_json("scylladb_upload.json")
 
+    def get_qa_users(self):
+        return self._get_json("qa_users.json")
```

Comment:
where do we get this qa_users.json from?

Reply:
This code download from our s3 bucket

Reply:
from the secure Keystore (S3 bucket). I created the file beforehand.